rero_ils/modules/entities/local_entities/proxy.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Local entity proxies."""
+from elasticsearch_dsl import Q
+
+from .api import LocalEntitiesSearch
+from ..models import EntityType
+
+
+CATEGORY_FILTERS = {
+    'agents': Q('terms', type=[EntityType.PERSON, EntityType.ORGANISATION]),
+    'person': Q('term', type=EntityType.PERSON),
+    'organisation': Q('term', type=EntityType.ORGANISATION),
+    'concepts': Q('term', type=EntityType.TOPIC),
+    'concepts-genreForm':
+        Q('term', type=EntityType.TOPIC) & Q('term', genreForm=True)
+}
+
+
+class LocalEntityProxy:
+    """Local entity proxy."""
```

Comment:
I know the context, but from my point of view, it is not really a proxy.
I think this is better to do that in search class.